ros_gz_bridge/README.md

```diff
@@ -28,17 +28,18 @@ The following message types can be bridged for topics:
 | geometry_msgs/msg/TransformStamped   | ignition::msgs::Pose                   |
 | geometry_msgs/msg/Twist              | ignition::msgs::Twist                  |
 | geometry_msgs/msg/TwistWithCovariance| ignition::msgs::TwistWithCovariance    |
-| mav_msgs/msg/Actuators (TODO)        | ignition::msgs::Actuators (TODO)       |
 | nav_msgs/msg/Odometry                | ignition::msgs::Odometry               |
 | nav_msgs/msg/Odometry                | ignition::msgs::OdometryWithCovariance |
-| ros_gz_interfaces/msg/Contact       | ignition::msgs::Contact                |
-| ros_gz_interfaces/msg/Contacts      | ignition::msgs::Contacts               |
-| ros_gz_interfaces/msg/Entity        | ignition::msgs::Entity                 |
-| ros_gz_interfaces/msg/GuiCamera     | ignition::msgs::GUICamera              |
-| ros_gz_interfaces/msg/JointWrench   | ignition::msgs::JointWrench            |
-| ros_gz_interfaces/msg/Light         | ignition::msgs::Light                  |
-| ros_gz_interfaces/msg/StringVec     | ignition::msgs::StringMsg_V            |
-| ros_gz_interfaces/msg/TrackVisual   | ignition::msgs::TrackVisual            |
+| rcl_interfaces/msg/ParameterValue    | ignition::msgs::Any                  |
```

Comment:
instead of `ignition` is `gz` ?

Reply:
For the purpose of `humble`, we need to still maintain support for older versions with the previous set of `ignition` names. 

I will have a follow-up PR for the final renaming bits, but that will make it so that whatever branch it lands on will only be compatible with `garden` forward.

Reply:
+1 to this, yep!